config/application.rb

```diff
@@ -38,3 +38,5 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :sidekiq
   end
 end
+
+require_relative "../lib/digested_assets_path_resolver"
```

Comment:
zeitwerk did this for us?

Reply:
I want to use `autoload_lib(ignore:)` when this app updates to v7.1.

https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#config-autoload-lib-ignore